easybuild/easyconfigs/p/PyQt5/PyQt5-5.12.1-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -42,25 +49,38 @@ components = [
         'options': {'modulename': 'PyQt5.sip'},
     }),
     (name, version, {
-        'source_urls': ['https://www.riverbankcomputing.com/static/Downloads/PyQt5/%(version)s/'],
+        'source_urls': ['https://www.riverbankcomputing.com/static/Downloads/%(name)s/%(version)s/'],
         'sources': ['%(name)s_gpl-%(version)s.tar.gz'],
         'checksums': ['3718ce847d824090fd5f95ff3f13847ee75c2507368d4cbaeb48338f506e59bf'],
-        'start_dir': 'PyQt5_gpl-%(version)s',
+        'start_dir': '%(name)s_gpl-%(version)s',
         'preconfigopts': "export PATH=%(installdir)s/bin:$PATH && ",
         'configopts': pyqt5_configopts,
         'options': {'modulename': 'PyQt5.QtCore'},
     }),
+    ('PyQtWebEngine', version, {
+        'source_urls': ['https://www.riverbankcomputing.com/static/Downloads/%(name)s/%(version)s/'],
+        'sources': ['%(name)s_gpl-%(version)s.tar.gz'],
+        'checksums': ['860704672ea1b616e1347be1f347bc1c749e64ed378370863fe209e84e9bd473'],
+        'start_dir': '%(name)s_gpl-%(version)s',
+        'preconfigopts': ("export PATH=%(installdir)s/bin:$PATH && "
+                          "export PYTHONPATH=%(installdir)s/lib/python%(pyshortver)s/site-packages:$PYTHONPATH && "),
+        'configopts': pyqtweb_configopts,
+        'options': {'modulename': 'PyQt5.QtWebEngine'},
+    }),
 ]
 
 sanity_check_paths = {
     'files': ['bin/sip', 'include/sip.h', 'lib/python%(pyshortver)s/site-packages/sipconfig.py',
               'lib/python%(pyshortver)s/site-packages/sipdistutils.py',
-              'lib/python%%(pyshortver)s/site-packages/PyQt5/sip.%s' % SHLIB_EXT],
+              'lib/python%%(pyshortver)s/site-packages/%%(name)s/sip.%s' % SHLIB_EXT,
+              'lib/python%%(pyshortver)s/site-packages/%%(name)s/QtCore.%s' % SHLIB_EXT,
+              'lib/python%%(pyshortver)s/site-packages/%%(name)s/QtWebEngine.%s' % SHLIB_EXT],
```

Comment:
this can be condensed to:
```
    'files': ['bin/sip', 'include/sip.h'] + ['lib/python%%(pyshortver)s/site-packages/%s' % x for x in (
        'sipconfig.py',
        'sipdistutils.py',
        '%%(name)s/sip.%s' % SHLIB_EXT,
        '%%(name)s/QtCore.%s' % SHLIB_EXT,
        '%%(name)s/QtWebEngine.%s' % SHLIB_EXT,
    )],
 ```

Reply:
That's neat :+1: